docs/001_develop/03_client-capabilities/008_interaction/number-field.mdx

```diff
@@ -0,0 +1,357 @@
+
+### Attributes
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Type</th>
+      <th>Description</th>
+      <th>Example</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>autofocus</td>
+      <td><code>boolean</code></td>
+      <td>When true, the component will be focused when the page has finished loading</td>
+<td>
+
+```typescript
+<rapid-number-field ?autofocus="${(x) => x.autoFocus}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>appearance</td>
+      <td><code>string</code></td>
+      <td>Can be <code>outline</code> or <code>filled</code></td>
+<td>
+
+```typescript
+<rapid-number-field appearance="outline">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>disabled</td>
+      <td><code>boolean</code></td>
+      <td>Similar to <code>readonly</code>, but with a blur on the component</td>
+<td>
+
+```typescript
+<rapid-number-field ?disabled="${(x) => x.isDisabled}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>locale</td>
+      <td><code>string</code></td>
+      <td>Defines a number format based on language and location. **Default: "en-US"**. Must be used with `withFormatting`</td>
+<td>
+
+```typescript
+<rapid-number-field locale="en-GB">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>form</td>
+      <td><code>string</code></td>
+      <td>Associates this component with a form. Form <code>id</code> needs to be passed. If no Id is provided, then it will be associated with the ancestor form</td>
+<td>
+
+```typescript
+<rapid-number-field form="myForm">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>hideStep</td>
+      <td><code>boolean</code></td>
+      <td>Hides the step control (up and down arrows) for the element</td>
+<td>
+
+```typescript
+<rapid-number-field hideStep>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>maxlength</td>
+      <td><code>number</code></td>
+      <td>The maximum number of characters allowed</td>
+<td>
+
+```typescript
+<rapid-number-field maxlength="${(x) => x.maxLength}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>minlength</td>
+      <td><code>number</code></td>
+      <td>The minimum number of characters allowed</td>
+<td>
+
+```typescript
+<rapid-number-field minlength="${(x) => x.minLength}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>max</td>
+      <td><code>number</code></td>
+      <td>Defines the maximum value allowed</td>
+<td>
+
+```typescript
+<rapid-number-field max="${(x) => x.max}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>min</td>
+      <td><code>number</code></td>
+      <td>Defines the minimum value allowed</td>
+<td>
+
+```typescript
+<rapid-number-field min="${(x) => x.min}">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>withFormatting</td>
+      <td><code>boolean</code></td>
+      <td>Enables you to format the number, use <code>maximumFractionDigits</code> to set the number of decimal places.</td>
+<td>
+
+```typescript
+<rapid-number-field withFormatting :options=${() => ({maximumFractionDigits: 2})}>
```

Comment:
It seems that `options` is a property from this line, so you need to move it to a table under that section instead of in the attribue table